src/PhpCs/FiveLab/ErrorCodes.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
-/*
+declare(strict_types = 1);
+
+/**
```

Comment:
It's wrong, because this is a PHP DocComment, but not simple comment. All licence blocks MUST write as simple comment. PHP Doc Comment MUST use only for classes, properties, methods.